packages/google_maps_flutter/google_maps_flutter_android/lib/src/google_maps_flutter_android.dart

```diff
@@ -1166,6 +1166,37 @@ PlatformJointType platformJointTypeFromJointType(JointType jointType) {
   return PlatformJointType.mitered;
 }
 
+/// Converts platform interface's Cap to Pigeon's PlatformCap.
```

Comment:
No action required: You are following the pattern of the rest of this file but both this method name and documentation do not help me understand what is being converted here or how to learn about either the to or from object.

Reply:
We could trivially improve them all a fair amount by using `[]` references for both class names. That was just an oversight on my part when I started converting.

Reply:
Please use dart doc linking notation for the classes.